snippets/freeipa_register.erb

```diff
@@ -24,12 +24,16 @@ name: freeipa_register
 #
 #   freeipa_opts                Additional options to pass directly to installer
 #
+
 <% if @host.operatingsystem.family == 'Redhat' -%>
   <% if @host.operatingsystem.name == 'Fedora' -%>
     freeipa_client=freeipa-client
   <% else -%>
     freeipa_client=ipa-client
   <% end -%>
+  <% if os_major > 6 -%>
```

Comment:
`os_major` won't/may not be defined here as it's not in this snippet. Use `@host.operatingsystem.major.to_i` or define it in the header.